transifex/requirements/edx-platform.txt

```diff
@@ -0,0 +1,11 @@
+Babel==1.3
+Django==1.10.7
```

Comment:
This should probably be the same version of Django used by edx-platform, which appears to be 1.8.18.

Reply:
ok